newsfragments/3143.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fall back to distutils install rather than failing.
```

Comment:
This news fragment on it's own makes no sense and gives no context. Consider:

```suggestion
With `setup.py install --prefix=...`, fall back to distutils install rather than failing. Note that running `setup.py install` is deprecated.
```

Reply:
Sounds good. The syntax needs to be rst, so I'll apply the change slightly differently.

```suggestion
With ``setup.py install --prefix=...``, fall back to distutils install rather than failing. Note that running ``setup.py install`` is deprecated.
```